src/ray/core_worker/common.h

```diff
@@ -70,7 +70,9 @@ struct ActorCreationOptions {
                        const std::unordered_map<std::string, double> &resources,
                        const std::unordered_map<std::string, double> &placement_resources,
                        const std::vector<std::string> &dynamic_worker_options,
-                       bool is_detached, std::string &name, bool is_asyncio)
+                       bool is_detached, std::string &name, bool is_asyncio,
+                       PlacementGroupID placement_group_id = PlacementGroupID::Nil(),
+                       int64_t bundle_index = -1)
```

Comment:
Can we typedef these together?
`typedef PlacementOptions std::pair<PlacementGroupID, int>`